src/nn/webgl/WebGLModel.js

```diff
@@ -689,6 +689,7 @@ export default class WebGLModel {
   }
 
   static _supportWebGL() {
+    tf.backend();
```

Comment:
what does this call do?

Reply:
In the latest version, `tf.getBackend()`  returns `undefined` if we do nothing before it. 

`tf.backend()` will register webgl backend if no backend has been registered.
Changed this to `tf.setBackend('webgl');` to be clearer.